web-ui/src/main/resources/catalog/templates/editor/directory.html

```diff
@@ -1,107 +1,304 @@
+                        &nbsp;<i class="caret"/>
+                      </button>
+                      <ul class="dropdown-menu pull-right" role="menu">
+                        <li><a href ng-click="startEditing(e)">
+                          <i class="fa fa-pencil fa-fw" />&nbsp;
+                          {{'edit' | translate}}</a>
+                        </li>
+                        <li><a href ng-click="copyEntry(e)">
+                          <i class="fa fa-copy fa-fw" />&nbsp;
+                          {{'duplicate' | translate}}</a>
+                        </li>
+                        <li><a href ng-click="startPermissionsEdit(e)">
+                          <i class="fa fa-key fa-fw" />&nbsp;
+                          {{'directoryEntryPermissions' | translate}}</a>
+                        </li>
+                        <li><a href ng-click="convertToTemplate(e)">
+                          <i class="fa fa-mail-forward fa-fw text-info" />&nbsp;
+                          {{'directoryEntryToTemplate' | translate}}</a>
+                        </li>
+                        <li><a href ng-click="validateEntry(e)">
+                          <i class="fa fa-check fa-fw text-success" />&nbsp;
+                          {{'directoryEntryValidate' | translate}}</a>
+                        </li>
+                        <li><a href ng-click="rejectEntry(e)">
+                          <i class="fa fa-ban fa-fw" />&nbsp;
+                          {{'directoryEntryReject' | translate}}</a>
+                        </li>
+                        <li><a href ng-click="delEntry(e)">
+                          <i class="fa fa-times fa-fw text-danger" />&nbsp;
+                          {{'delete' | translate}}</a>
+                        </li>
+                      </ul>
+                    </div>
+
+                    <div class="btn-group">
+                      <button class="btn btn-default btn-sm dropdown-toggle"
+                        data-toggle="dropdown">
+                        {{'directoryEntryAssociatedMetadata' |  translate}}
+                        &nbsp;<i class="caret"/>
+                      </button>
+                      <ul class="dropdown-menu pull-right" role="menu">
+                        <li><a href>
+                          placeholder</a>
+                        </li>
+                      </ul>
+                    </div>
+                  </div>
+
+                  <div class="entry-title entry-is-template"
+                    ng-if="templatesShown()">
+                    <span ng-class="e.valid == 1 ? 'text-success' : (md.valid == 0 ? 'text-danger' : 'text-muted')">
+                      <i class="fa gn-recordtype-t"
+                         title="{{('isTemplate' + e.isTemplate) | translate}} ({{('validStatus-' + e.valid) | translate}})"></i>
+                    </span>
+
+                    <a href ng-click="startEditing(e)" title="{{e.title}}">
```

Comment:
We also used to have a simple group filter (based on facet) 
![image](https://user-images.githubusercontent.com/1701393/27442602-f3ca4618-5770-11e7-98e2-6c4f12f74050.png)

which is quite useful when you have lots of entries. Eg. in Sextant, we have more than 10000 entries which are dispatched in different groups (to make search easier or limit to a subset of entries depending on the editor view).

We should probably keep that feature and maybe put in a different way in the UI.